openpype/client/server/operations.py

```diff
@@ -358,7 +358,10 @@ def prepare_hero_version_update_data(old_doc, new_doc, replace=True):
         Dict[str, Any]: Changes between old and new document.
     """
 
-    return _prepare_update_data(old_doc, new_doc, replace)
+    changes = _prepare_update_data(old_doc, new_doc, replace)
+    if "version_id" in changes:
+        changes.pop("version_id")
```

Comment:
Any reason not to just do this? Or is that slower?
```suggestion
    changes.pop("version_id", None)
```